nifi-nar-bundles/nifi-standard-services/nifi-hbase_1_1_2-client-service-bundle/nifi-hbase_1_1_2-client-service/src/main/java/org/apache/nifi/hbase/HBase_1_1_2_ClientService.java

```diff
@@ -171,6 +188,15 @@ protected Connection createConnection(final ConfigurationContext context) throws
             }
         }
 
+        UserGroupInformation.setConfiguration(hbaseConfig);
+
+        if (UserGroupInformation.isSecurityEnabled()) {
+            LOG.info("SECURITY IS ENABLED");
+            UserGroupInformation.loginUserFromKeytab(context.getProperty(KERBEROS_PRINCIPAL).getValue(),
```

Comment:
After the initial login, you'll need to periodically renew your ticket. See [AbstractHadoopProcessor.java](https://github.com/apache/nifi/blob/master/nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/AbstractHadoopProcessor.java#L425-L439) for an example.